src/App.tsx

```diff
@@ -28,15 +30,12 @@ const App: React.FC = () => {
               <Switch>
                <Route exact path="/"
                   render={props => {
-                    return <>
-                      <p>There are a few main entrypoints into this web app</p>
-                      <p>
-                        <a href='/register1'>Account Registration</a>,&nbsp;
-                        <a href='/authenticated/validate'>Profile Validation</a>,&nbsp;and&nbsp;
-                        <a href='/authenticated/myaccount'>My Account</a>
-                      </p>
-                      </>
+                    return <Redirect to='/authenticated/myaccount' />
```

Comment:
Got rid of the unstyled landing page, and redirect to My Account.  (Note that I updated the readme to show the available entry points into this application).